src/controllers/whototell.controller.ts

```diff
@@ -0,0 +1,50 @@
+import { BAD_REQUEST, OK } from 'http-status-codes'
+import { inject } from 'inversify'
+import { controller, httpGet, httpPost, requestBody } from 'inversify-express-utils'
+import BaseController from './base.controller'
+
+import Optional from 'app/models/optional'
+import ValidationErrors from 'app/models/validationErrors'
+import { WhototellFormModel } from 'app/models/whototell.model'
+import { ROOT_URI, SEARCH_COMPANY_URI, WHO_TO_TELL_URI } from 'app/paths'
+import formSchema from 'app/schemas/whototell.schema'
+import FormValidator from 'app/utils/formValidator.util'
+
+interface ViewModel {
+  backUri?: string
+  data?: WhototellFormModel
+  errors?: ValidationErrors
+}
+
+@controller(WHO_TO_TELL_URI)
+export class WhototellController extends BaseController {
```

Comment:
Please consider naming it WhoToTellController

Reply:
+1